test_package/soci_mysql_backend.cpp

```diff
@@ -1,20 +1,17 @@
 #include <soci/soci.h>
 #include <soci/mysql/soci-mysql.h>
 
-#include <stdexcept>
-
 int main() {
-    soci::session sql(soci::mysql,
```

Comment:
AppVeyor https://ci.appveyor.com/project/tt4g/conan-soci/builds/21584193/job/867heus68xixrju6

error LNK2019: unresolved external symbol "struct soci::mysql_backend_factory const soci::mysql.

Reply:
I'm found.

http://soci.sourceforge.net/doc/master/connections/

>Portability note
> In case of SOCI linked against DLLs on Windows, the factory objects are not exported from the DLLs. In order to avoid linker errors, access factory objects via dedicated backend functions provided (eg. factory_postgresql()).

Reply:
Fixes 958808d994e48026d4a28e594e029cabf1c5b4db